testsite/foo/templates/index.html

```diff
@@ -3,19 +3,32 @@
 <html>
 <head>
     <title>Test Civet</title>
-    <link rel="stylesheet" href="{% static 'global.css' %}">
-    <link rel="stylesheet" href="{% static 'shared.css' %}">
-    <link rel="stylesheet" href="{% static 'local.css' %}">
-    <link rel="stylesheet" href="{% static 'library.css' %}">
-    <script src="{% static 'global.js' %}" type="text/javascript" charset="utf-8" async defer></script>
-    <script src="{% static 'shared.js' %}" type="text/javascript" charset="utf-8" async defer></script>
-    <script src="{% static 'local.js' %}" type="text/javascript" charset="utf-8" async defer></script>
-    <script src="{% static 'library.js' %}" type="text/javascript" charset="utf-8" async defer></script>
+    <!-- should be found -->
+    <link rel="stylesheet" href="{% static 'css/global.css' %}">
+    <link rel="stylesheet" href="{% static 'sass/shared.css' %}">
+    <link rel="stylesheet" href="{% static 'css/local.css' %}">
+    <link rel="stylesheet" href="{% static 'sass/library.css' %}">
+
+    <!-- should produce error -->
+    <link rel="stylesheet" href="{% static 'some_managed_package/test.css' %}">
+    <link rel="stylesheet" href="{% static 'bower_components/test/test_styles/main.css' %}">
```

Comment:
> main.css actually loads after you've run bower install, doesn't it?

Ah, yes, but this is tricky.  `_sass_partial.scss` defines `color: blue;` for `.foo`, but because Civet ignores those files text continues to display white.  It just serves as further proof that files in that dir are ignored by Civet.

I'll update the HTML comments accordingly.